README.md

```diff
@@ -5,7 +5,7 @@ Kubernetes project infrastructure, managed by the kubernetes community via [sig-
 - `apps`: community-managed apps that run on the community-managed `aaa` cluster
     - `codesearch`: instance of [codesearch] at https://cs-canary.k8s.io - owned by [sig-k8s-infra]
     - `elekto`: instance of [elekto] at https://elections.k8s.io - owned by Elections officers (on behalf of [sig-contributor-experience])
-    - `gcsweb`: instance of [gcsweb] at https://gcsweb.k8s.io - owned by [sig-testing]
+    - `gcsweb`: instance of [gcsweb] at https://pkg.go.dev/k8s.io/test-infra/gcsweb - owned by [sig-testing]
```

Comment:
The link is definitely broken, but is that the intention of that link?

Reply:
Just remove the https://, it's a reference to what domain we've deployed this at, it is in fact running at gcsweb.k8s.io, it just doesn't have a nice landing page.

It's a tiny service that renders pages like https://gcsweb.k8s.io/gcs/kubernetes-jenkins/pr-logs/pull/enhancements/4092/pull-enhancements-test/1669106952459784192/

TBH we should probably fold this into prow.k8s.io, it's just not a priority for anyone.